### ISSUE TITLE
Add VPN service, allow VPN connections to Dexter

### ISSUE THREAD

**Issue by JamesNewton** [HaddingtonDynamics/Dexter#80]
Since Windows 10 up no longer reliably allows connection to SAMBA shares, #58 setting up a VPN server on Dexter to re-enable access to that may be a more attractive option than [SFTP](https://github.com/HaddingtonDynamics/Dexter/wiki/Dexter-Networking#sftp). 

There are many options. "The wonderful thing about Standards is that there are so many from which to choose."

[OpenVPN](https://github.com/OpenVPN/openvpn) is the obvious choice. It is open source, and older versions work on 16.04 but [it does NOT work with the existing VPN built into Windows](https://serverfault.com/questions/830919/connect-to-openvpn-using-windows-10-built-in-vpn), so users would have to be able to install OpenVPN client on their PCs. While that may be possible, avoiding the requirement seems worthwhile. If Windows users will have to install something, they might as well install an [SFTP client](https://github.com/HaddingtonDynamics/Dexter/wiki/Dexter-Networking#sftp). 

The Windows built-in VPN client supports only IKEv2, L2TP, PPTP and SSTP tunneling protocols.

IKEv2 was developed by M$ and Cisco and [isn't bad](https://www.cactusvpn.com/beginners-guide-to-vpn/what-is-ikev2/), it's difficult to [setup on Ubuntu 16.04](https://www.digitalocean.com/community/tutorials/how-to-set-up-an-ikev2-vpn-server-with-strongswan-on-ubuntu-16-04). While most OS's support connecting to an IKEv2 VPN, the setup is again difficult on non M$ OSs. 

L2TP/IPSec can probably be supported via [Openswan and xl2tpd](https://github.com/xelerance/Openswan/wiki/L2tp-ipsec-configuration-using-openswan-and-xl2tpd). Most tutorials assume that the Ubuntu server is public facing or will be accessed through a firewall. Some changes may be necessary for local access. 

PPTP is old, often disabled by routers, and inherently insecure, but since we are doing this on a local network with a client that is totally insecure anyway, who cares? It's [very easy to setup](https://bobcares.com/blog/install-pptp-server-ubuntu/), fast, and still supported by every client. This is probably the best bet for an initial setup.
 
SSTP is M$ proprietary and while there are services available for Ubuntu, it is not well support by clients other than Windows.

**Comment by JamesNewton** [comment]
Tried the very easy setup from 
https://bobcares.com/blog/install-pptp-server-ubuntu/
and on step 6 I got this error:
`libkmod: ERROR ../libkmod/libkmod-config.c:635 kmod_config_parse: /etc/modprobe.d/pptpd.conf line 1: ignoring bad line starting with 'nf_nat_pptp'`
It may not be important:
https://bugs.launchpad.net/ubuntu/+source/pptpd/+bug/1571295
or can be worked around.

I am able to authenticate (no surprise) but after "Registering your computer on the network" I get 
`Error 734: The PPP link control protocol was terminated`

`systemctl status pptpd` on Dexter shows that Windows refused to authenticate? 

I enabled CHAP and CHAP2 authentication on the windows side and now I'm getting an error message about not having a CHAP secret. 

Troubleshooting is needed.
Just noticed those instructions are not 16.04 specific, but at first glance:
https://sysadmins.co.za/setup-a-pptp-vpn-on-ubuntu-16/
appears to be the same.
